### DISCUSSION TITLE
Is canceling delayed publish released and how can I use it?

### DISCUSSION THREAD

**Discussion by t2wu** [emqx/emqx#8011] [General]
Hi I noticed that there seemed to be a branch for canceling delayed publish. Is the feature released yet? And if so, is there any documentation on how I can use it?

**Comment by zmstone** [comment]
Hi
If the code is not merged into main-v4.3 or main-v4.4 it’s likely not released.
could you let us know the branch name?

**Reply by t2wu** [reply]
I forgot if I've actually seen the branch or not. But there seems to be a reference to it [here](https://github.com/emqx/emqx/issues/4984). I am not sure what "down" means by DDDHuang.
